.circleci/config.yml

```diff
@@ -578,7 +578,9 @@ workflows:
           filters:
             branches:
               only:
+                - production
                 - staging
+                - integration
```

Comment:
Je ne pense que rebuilder l'image soit necessaire. Surtout qu'on va la tagger avec le meme tag de version qu'en staging

Reply:
Pas trouvé mieux que de mettre ce IF dans le job...
Temps d'echec `build-and-push-image` jusqu'au if ~70s